ansible_collections/arista/avd/roles/eos_designs/templates/base/base.py

```diff
@@ -0,0 +1,658 @@
+                    # updating for cvaas_ips
+                    daemon_terminattr['cvaddrs'].append(f"{cvp_instance_ip}:443")
+                    daemon_terminattr['cvauth'] = {
+                        'method': 'token-secure',
+                        'token_file': get(self._hostvars, 'cvp_token_file', '/tmp/cv-onboarding-token'),
+                    }
+                else:
+                    # updating for cvp_on_prem_ips
+                    cv_address = f"{cvp_instance_ip}:{get(self._hostvars, 'terminattr_ingestgrpcurl_port')}"
+                    daemon_terminattr['cvaddrs'].append(cv_address)
+                    daemon_terminattr['cvauth'] = {
+                        'method': 'key',
+                        'key': get(self._hostvars, 'cvp_ingestauth_key'),
+                    }
+        daemon_terminattr['cvvrf'] = self._mgmt_interface_vrf
+        daemon_terminattr['smashexcludes'] = get(self._hostvars, 'terminattr_smashexcludes')
+        daemon_terminattr['ingestexclude'] = get(self._hostvars, 'terminattr_ingestexclude')
+        daemon_terminattr['disable_aaa'] = get(self._hostvars, 'terminattr_disable_aaa', False)
+        return daemon_terminattr
+
+    @cached_property
+    def vlan_internal_order(self):
+        """
+        vlan_internal_order set based on internal_vlan_order data-model
+        """
+        return {
+            'allocation': get(self._hostvars, 'internal_vlan_order.allocation'),
+            'range': {
+                'beginning': get(self._hostvars, 'internal_vlan_order.range.beginning'),
+                'ending': get(self._hostvars, 'internal_vlan_order.range.ending')
+            }
+        }
+
+    @cached_property
+    def event_monitor(self):
+        """
+        event_monitor set based on event_monitor data-model
+        """
+        if get(self._hostvars, 'event_monitor') is not True:
+            return None
+        return {'enabled': 'true'}
+
+    @cached_property
+    def event_handlers(self):
+        """
+        event_handlers set based on event_handlers data-model
+        """
+        if (event_handlers := get(self._hostvars, 'event_handlers')) is None:
+            return None
+        event_handlers = self._convert_dicts(event_handlers, 'name')
+        event_handler = {}
+        for handler in event_handlers:
+            handler_name = get(handler, 'name')
+            event_handler[handler_name] = {}
+            action = handler.get('action')
+            action_type = handler.get('action_type')
+            if action is not None and action_type is not None:
+                event_handler[handler_name]['action_type'] = action_type
+                event_handler[handler_name]['action'] = action
+            if (delay := handler.get('delay')) is not None:
+                event_handler[handler_name]['delay'] = delay
+            if get(handler, 'asynchronous') is True:
+                event_handler[handler_name]['asynchronous'] = 'handler.asynchronous'
```

Comment:
This was a bug in the old code. This should just be the value of the input value like all the others.